src/test/java/org/opensearch/security/securityconf/SecurityRolesPermissionsV6Test.java

```diff
@@ -1,193 +0,0 @@
-/*
- * SPDX-License-Identifier: Apache-2.0
- *
- * The OpenSearch Contributors require contributions made to
- * this file be licensed under the Apache-2.0 license or a
- * compatible open source license.
- *
- * Modifications Copyright OpenSearch Contributors. See
- * GitHub history for details.
- */
-
-package org.opensearch.security.securityconf;
-
-import java.io.IOException;
-import java.util.Arrays;
-import java.util.List;
-import java.util.Map;
-import java.util.TreeMap;
-
-import com.google.common.collect.ImmutableMap;
-import com.google.common.collect.ImmutableSet;
-import com.fasterxml.jackson.databind.ObjectMapper;
-import com.fasterxml.jackson.databind.node.ArrayNode;
-import com.fasterxml.jackson.databind.node.ObjectNode;
-import org.junit.Assert;
-import org.junit.Test;
-
-import org.opensearch.action.support.IndicesOptions;
-import org.opensearch.cluster.ClusterState;
-import org.opensearch.cluster.metadata.IndexAbstraction;
-import org.opensearch.cluster.metadata.IndexNameExpressionResolver;
-import org.opensearch.cluster.metadata.Metadata;
-import org.opensearch.cluster.service.ClusterService;
-import org.opensearch.common.settings.Settings;
-import org.opensearch.security.DefaultObjectMapper;
-import org.opensearch.security.resolver.IndexResolverReplacer;
-import org.opensearch.security.securityconf.impl.CType;
-import org.opensearch.security.securityconf.impl.SecurityDynamicConfiguration;
-import org.opensearch.security.support.ConfigConstants;
-import org.opensearch.security.user.User;
-
-import org.mockito.quality.Strictness;
-
-import static org.mockito.Mockito.doReturn;
-import static org.mockito.Mockito.mock;
-import static org.mockito.Mockito.when;
-import static org.mockito.Mockito.withSettings;
-
-public class SecurityRolesPermissionsV6Test {
```

Comment:
why delete v6 test?

Reply:
Now we convert "behind the scenes" any V6 config into a V7 config. Thus, the business logic which operates on the V6 config is no longer in use - that's especially the classes `ConfigModelV6` and `DynamicConfigModelV6`. I actually forgot to delete these classes in the first commit - in a second commit, I have now deleted these classes.

As the `SecurityRolesPermissionsV6Test` tests the business logic in the  `ConfigModelV6`  class, it has now nothing to test anymore.